webapp/test/mock/resultset_list.json

```diff
@@ -3,8 +3,7 @@
     "count": 10,
     "filter_params": {
       "count": "10",
-      "full": "true",
-      "format": "json"
+      "full": "true"
```

Comment:
``full`` defaults to true in the api, so we can probably remove it anywhere it's not set to false.